configure.py

```diff
@@ -240,24 +263,24 @@ def setup_python(environ_cp):
   environ_cp['PYTHON_BIN_PATH'] = python_bin_path
 
   # Write tools/python_bin_path.sh
-  with open('tools/python_bin_path.sh', 'w') as f:
+  with open(os.path.join(_TF_WORKSPACE, 'tools/python_bin_path.sh'), 'w') as f:
```

Comment:
Looks like running `bazel run //:configure` from tf repo won't work because of this file. Is there any way around this?